solvation_analysis/tests/test_analysis.py

```diff
@@ -0,0 +1,50 @@
+import matplotlib.pyplot as plt
+import pytest
+
+from copy import deepcopy
+
+from solvation_analysis.analysis import Solution
+import numpy as np
+
+
+def test_plot_solvation_distance(rdf_bins_and_data_easy):
+    bins, data = rdf_bins_and_data_easy['pf6_all']
+    fig, ax = Solution._plot_solvation_radius(bins, data, 2)
+    # plt.show()  # comment out for global testing
+
+
+def test_instantiate_solute(default_solute):
+    assert len(default_solute.radii) == 1
+    assert len(default_solute.rdf_data) == 0
+    assert len(default_solute.rdf_plots) == 0
+    assert callable(default_solute.kernel)
+    assert default_solute.solute.n_residues == 49
+    assert default_solute.solvents['pf6'].n_residues == 49
+    assert default_solute.solvents['fec'].n_residues == 237
+    assert default_solute.solvents['bn'].n_residues == 363
+
+
+def test_radii_finding(run_solute):
+    assert len(run_solute.radii) == 3
+    assert len(run_solute.rdf_data) == 3
+    assert len(run_solute.rdf_plots) == 3
+    assert 2 < run_solute.radii['pf6'] < 3
```

Comment:
This took me longer than it should have to figure out. A little comment will help.